flow-typed/npm/downshift_v2.x.x.js.flow

```diff
@@ -114,27 +114,97 @@ declare module downshift {
   declare export type StateChangeFunction<Item> = (
     state: DownshiftState<Item>,
   ) => StateChangeOptions<Item>
+
+  declare export type GetRootPropsReturn = {
+    role: 'combobox';
+    // 'aria-expanded': boolean;
```

Comment:
Flow doesn't like properties declared as strings

Reply:
That's pretty unfortunate and surprising as well 🤔

Reply:
I'm not seeing that in the REPL? https://flow.org/try/#0C4TwDgpgBAKhDOwoF4oG8BQUoEgBGA-AFxR4D2ZANhAIYB2A3BjgOQC2IAtHQK5t4QATixK9+QpgF8MGAMZk6iKDRJwlqNFHZcxA4SQDMUSUyA